### PR TITLE
re:tools

### PR DESCRIPTION
**Pull request by triplezeta**
## About The Pull Request

this resprites some tool sprites. again.

Azlan's (AzlanonPC, Azlan#0852, someangryminer, etc.)  screwdriver sprite has been taken and modified to replace our current one.
image

multitool and geiger counter have been given fresh, vibrant palettes instead of the dull desaturated ones from before.
image

a lot of unique inhands have been added for things that really should've gotten unique inhands, such as the pipe wrench and the syndicate tool variants (except for the screwdriver and wirecutters   and also welder tools), along with the geiger and jaws of life inhands being fixed.
image

T-ray scanner looks a bit different.
image

Analyzer has been returned to its original shape, but heavily desouled. 
image

additional changes: ancient multitool variant added, "cobbyed" added as an attack verb for the medical wrench

## Why It's Good For The Game

soul b gone

## Changelog
:cl:
imageadd: inhands have been fixed for some tools, added and/or desouled for others.
imageadd: Azlan's screwdriver sprite
imageadd: multitool, geiger counter, t-ray scanner, gas analyzer, have all received new sprites
imagedel: my old screwdriver sprites
/:cl:

### CODE REVIEW

**Comment by trerri** [comment]
the right bit on the new analyzer sucks, the reds dont work at all and it looks downright blurry

File: code/game/objects/items/tools/wrench.dm [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /obj/item/wrench
 	name = "wrench"
-	desc = "A wrench with common uses. Can be found in your hand."
+	desc = "A wrench with common uses. Can be found in your hand. This one is a pipe wrench."
 	icon = 'waspstation/icons/obj/tools.dmi'
-	icon_state = "wrench"
-	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
-	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	icon_state = "wrench_pipe" //now where could my pipe wrench be?
```
Comment (MarkSuckerberg) [inline code/game/objects/items/tools/wrench.dm:5]: GARFIELD

File: waspstation/code/game/objects/items/devices/multitool.dm [hunk 2]
```diff
@@ -3,3 +3,9 @@
 	desc = "A darkened multitool with a matte finish and an ominous glowing screen."
 	icon_state = "multitool_syndie"
 	toolspeed = 0.5
+
+/obj/item/multitool/old
+	desc = "Used for pulsing wires to test which to cut. This one looks.. 'retro'. It wasn't recommended by doctors then and won't be recommended by doctors now."
```
Comment (MarkSuckerberg) [inline waspstation/code/game/objects/items/devices/multitool.dm:8]:
```suggestion
	desc = "Used for pulsing wires to test which to cut. This one looks... 'retro'. It wasn't recommended by doctors then and won't be recommended by doctors now."
```

File: waspstation/code/game/objects/items/devices/multitool.dm [hunk 3]
```diff
@@ -3,3 +3,9 @@
 	desc = "A darkened multitool with a matte finish and an ominous glowing screen."
 	icon_state = "multitool_syndie"
 	toolspeed = 0.5
```
Comment (Superyodeler) [inline waspstation/code/game/objects/items/devices/multitool.dm:5]: Looks like it's just using the default multitool's inhands.

**Review by Superyodeler — CHANGES_REQUESTED** [review]
Sprites look good, but the syndie multitool is using the regular multitool's inhand sprites.

File: waspstation/code/game/objects/items/devices/multitool.dm (hunk 3 above)
Comment (triplezeta) [reply waspstation/code/game/objects/items/devices/multitool.dm:5]: fuck

**Review by Superyodeler — APPROVED** [review]

**Pull request merged**